kea-vscode/src/views/common/file-tree-node.ts

```diff
@@ -13,13 +14,14 @@ export class FileTreeNode implements IParentTreeNode<ReviewCommentTreeNode> {
   #iconPath = new vscode.ThemeIcon("file");
```

Comment:
The `iconPath` for nodes which aren't the `LocalFileTreeNode` aren't the icons for the file extension - e.g., TypeScript files don't have the TS logo.